### ISSUE TITLE
Scheduled actions disabled after 60 days

### ISSUE THREAD

**Issue by mgcth** [Ingenjorsarbete-For-Klimatet/ingenjorsarbete-for-klimatet.github.io#71]
## Description

Scheduled actions are disabled after 60 days of no repo activity. That's why the last PR wasn't built automatically. We could use something like, https://github.com/marketplace/actions/keepalive-workflow, but that creates dummy commits. I think we could have two separate actions, one for the scheduled build and one based on merged pull requests.

**Comment by docNord** [comment]
It seems there is a scheduled build at 00:00 every night now? Does that mean this issue should be closed @mgcth ?

**Comment by mgcth** [comment]
No, the scheduling is disabled after 60 days and so is a manual merge to main build.
